.github/test-infra/aws/rke2/iam.tf

```diff
@@ -77,6 +77,20 @@ data "aws_iam_policy_document" "aws_ccm" {
   }
 }
 
+data "local_file" "helm_template" {
+  filename = "./scripts/helmchart-template.yaml"
+}
+
+data "http" "aws-lb-controller-iam" {
+  url = "https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/${split(":", yamldecode(local.aws_load_balancer_controller_template[4]).image)[1]}/docs/install/iam_policy_us-gov.json"
```

Comment:
This (and the supporting yaml/split logic) feel incredibly clunky and error-prone compared to just renovate-ing this directly? There is already a matcher in our common renovate config for this type of string, the only necessary change is [adding support for TF file matching](https://github.com/defenseunicorns/uds-common/pull/464). Seems like this would be a simpler approach?